hms-app/src/components/HackathonSelectDropdown.tsx

```diff
@@ -52,7 +63,7 @@ export default function HackathonSelectDropdown({
         return hackathonList
           .filter(
             (hackathon) =>
-              hackathon.endDate < today && hackathon.endDate > VALID_DATE
+              hackathon.endDate < today && hackathon.endDate > MIN_DATE
```

Comment:
why exactly do we do this check? do we expect there to be dates older than that? or do we want to have a configurable MIN_DATE that we can set to "hide" old hackathons?

Reply:
yeah so the min date hides the ideation portal hackathon and the max dates cutes the date out of the dropdown

Reply:
ok, it's just unclear as to what this does at first glance.  I get it all now though.  There are better ways we could do this but for now we can leave it